packages/react-ui-components/src/SelectBox/selectBox.js

```diff
@@ -158,40 +156,50 @@ export default class SelectBox extends PureComponent {
 
     componentDidUpdate() {
         const listener = e => {
```

Comment:
Is it needed to define a listener function on each update? Maybe set it as a class method or move it outside of the class declaration altogether?